routes/create.js

```diff
@@ -1,18 +1,51 @@
 const express = require("express");
+const Story = require("../models/Story");
+const User = require("../models/User");
+const Chapter = require("../models/Chapter");
 const router = express.Router();
 
 // @desc Create story page
 // @route POST /create
 // @access Private
 router.post("/", (req, res) => {
-  res.send("it should take book details"); // this should return back a storyID which will be used to create the chapter 
+  router.post("/", async(req, res) => {
```

Comment:
Why is the route declared two times?
It's not needed. 
Also add the /story to route to specify it's to create a astory route
```suggestion
  router.post("/story", async(req, res) => {
```